.changeset/frank-pans-accept.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/server': patch
+---
+
+Experiment item results now accept an optional `notScorable` field on each scorer result, set when the scorer declared the run not scorable.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)